nmrfx-processor/src/main/java/org/nmrfx/processor/optimization/Lmder_f77.java

```diff
@@ -407,21 +407,18 @@ public void doMin() {
                 }
             }
 
-            //System.out.println("Initial rmsd: " + rmsd1 + "Final rmsd: " + rmsd2+" Polish rmsd: "+rmsd3+"nfev "+nfev+" "+info[1]);
             na[ic] = n;
             ma[ic] = m;
             nf[ic] = nfev;
             na[ic] = info[1];
 
             fnm[ic] = fnorm2;
 
-            //System.out.print("\n Initial L2 norm of the residuals: " + fnorm1 +
-            //"\n Final L2 norm of the residuals: " + fnorm2 +
-            //"\n Number of function evaluations: " + nf[ic] +
-            //"\n Number of Jacobian evaluations: " + nj[ic] +
-            //"\n Info value: " + info[1] +
-            //"\n Final approximate solution: \n\n");
-            //tol /= Math.pow(1.0e4,1.0/ntries);
+            // Note: variable meanings are Initial L2 norm of the residuals: fnorm1
+            // Final L2 norm of the residuals: fnorm2
+            // Number of function evaluations: nf[ic]
+            // Number of Jacobian evaluations: nj[ic]
```

Comment:
Removed System.out here, but left the rest as a note in case the variable explanations were still needed